Week06/src/MainInventory.java

```diff
@@ -0,0 +1,152 @@
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Scanner;
+
+public class MainInventory {
+
+    public static void main(String[] args) {
+
+        Scanner sc = new Scanner(System.in);
+
+        System.out.println("Welcome! Choose the number from the options below to continue:");
+        System.out.println("1. Create product and add it to stock");
+        System.out.println("2. Sell product");
+        System.out.println("3. Display daily sales and report");
+        System.out.println("4. Exit");
+
+        List<Product> ProductList = new ArrayList<>();
+        List<Order> OrderList = new ArrayList<>();
```

Comment:
I would have liked these to be attributes of an object that manages the store...